### ISSUE TITLE
Allow changing of the doctrine dbs key

### ISSUE THREAD

**Issue by mathewpeterson** [bernardphp/silex#3]
Currently, the 'bernard' dbs key is hard coded, this should be configurable.

**Comment by henrikbjorn** [comment]
Any specific reason why?

**Comment by mathewpeterson** [comment]
Yes, it seems unnecessary to create a separate connection for the doctrine driver. I would have thought it would try and use the default connection.

**Comment by henrikbjorn** [comment]
You are free to do a PR and ill look into it :)
